changelog.d/14149.bugfix

```diff
@@ -0,0 +1 @@
+Fix #12383: paginate room messages from the start if no from is given. Contributed by @gnunicorn .
```

Comment:
```suggestion
Fix a long-standing bug where paginating forward in the room would start from the last event rather than the first, thus not giving any events. Contributed by @gnunicorn.
```

I think this is an accurate description? We don't refer to issues here; interested people can click through to the PR where you've linked the issue :)